uk.co.compucorp.civicrm.hrleaveandabsences/views/absence-tab/components/absence-tab-custom-work-pattern-modal.html

```diff
@@ -46,7 +46,7 @@ <h2 class="modal-title" id="myModalLabel">Custom Work Pattern</h2>
           <div class="form-group">
             <label class="col-sm-4 control-label">Reason for change</label>
             <div class="col-sm-8">
-              <div class="crm_custom-select crm_custom-select--full">
+              <div class="crm_custom-select crm_custom-select--medium">
```

Comment:
Not quite sure, please have a look how it is done for TOIL. You should consider mobile layout as well

https://github.com/civicrm/civihr/blob/staging/uk.co.compucorp.civicrm.hrleaveandabsences/views/shared/components/leave-request-popup/details-tab/toil.html